build.bat

```diff
@@ -8,7 +8,7 @@ if [%VisualStudioVersion%]==[] (
 if [%1]==[] goto USAGE
```

Comment:
This should jump to `USAGE` if `%2` is unset as well. Also, the "usage" text should be updated.

Reply:
Good catch!